plugins/azure/containerapps/containerAppHttpsOnly.js

```diff
@@ -0,0 +1,56 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Container Apps HTTPS only',
+    category: 'Container Apps',
+    domain: 'Containers',
+    severity: 'Medium',
+    description: 'Ensures Container Apps are only accessible over HTTPS.',
+    more_info: 'Using HTTPS guarantees that server/service authentication is established and shields data during transmission from potential network eavesdropping. Disabling allowInsecure triggers automatic redirection of HTTP requests to secure HTTPS connections, ensuring container apps are securely accessed.',
+    recommended_action: 'Modify Container app and disable allowInsecure setting in Ingress',
```

Comment:
Maybe we can write something like 'Enable HTTPS only by disabling allowInsecure feature for all Container apps.'